flink-ml-parent/flink-ml-lib/src/test/java/org/apache/flink/ml/common/utils/OutputColsHelperTest.java

```diff
@@ -0,0 +1,261 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.common.utils;
+
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.types.Row;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit test for OutputColsHelper.
+ */
+public class OutputColsHelperTest {
+
+	private TableSchema tableSchema = new TableSchema(
+		new String[]{"f0", "f1", "f2"},
+		new TypeInformation[]{
+			TypeInformation.of(String.class),
+			TypeInformation.of(Long.class),
+			TypeInformation.of(Integer.class)
+		}
+	);
+	private String[] reservedColNames = new String[]{"f0"};
+	private Row row = Row.of("a", 1L, 1);
+
+	@Test
+	public void testResultSchema() {
+		TableSchema expectSchema = new TableSchema(
+			new String[]{"f0", "f1", "f2", "res"},
+			new TypeInformation[]{
+				TypeInformation.of(String.class),
+				TypeInformation.of(Long.class),
+				TypeInformation.of(Integer.class),
+				TypeInformation.of(String.class)
+			}
+		);
+		OutputColsHelper helper = new OutputColsHelper(
+			tableSchema, "res",
+			TypeInformation.of(String.class)
+		);
+		Assert.assertEquals(helper.getResultSchema(), expectSchema);
```

Comment:
@xuyang1706 according to the tradition of `junit`, the expected value should be the first in the `assertXXX` calls. could you please fix this? 
See: http://junit.sourceforge.net/javadoc/org/junit/Assert.html

Reply:
Thanks, fixed.